pkg/config/log.go

```diff
@@ -86,7 +84,7 @@ func SetupLogger(logLevel, logFile, uri string, rfc, tls bool, pem string, logTo
 
 	configTemplate += `</formats>
 	</seelog>`
-	config := fmt.Sprintf(configTemplate, seelogLogLevel, logFile, logFileMaxSize, logDateFormat, logDateFormat)
+	config := fmt.Sprintf(configTemplate, seelogLogLevel, logDateFormat, logDateFormat)
```

Comment:
additional improvement we could make here, if you have time: formatting these last 3 items earlier, currently it's very hard to understand where in the `configTemplate` string these are supposed to be used.

Feel free to merge without addressing this though :) We could plan work on refactoring this whole function, which is becoming a bit of a mess.

Reply:
Done !